clients/privacy-center/config/config.json

```diff
@@ -30,21 +30,38 @@
   ],
   "includeConsent": true,
   "consent": {
+    "cookieName": "fides_consent",
     "consentOptions": [
       {
+        "name": "Advertising",
+        "fidesDataUseKey": "advertising",
+        "url": "https://example.com/privacy#advertising",
+        "description": "...",
+        "cookieKeys": ["data_sales"]
+      },
+      {
+        "name": "Analytics",
         "fidesDataUseKey": "third_party_sharing",
-        "name": "Do not sell my personal information",
-        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc vulputate libero et velit. Lorem ipsum dolor sit amet, consectetur.",
-        "highlight": true,
-        "url": "https://example.com/privacy#data-sales"
+        "description": "...",
+        "url": "https://example.com/privacy#analytics",
+        "cookieKeys": ["data_sharing"]
       },
       {
+        "name": "Personalize",
+        "fidesDataUseKey": "personalize",
         "default": true,
+        "url": "https://example.com/privacy#personalize",
+        "description": "...",
+        "cookieKeys": ["functional"]
+      },
+      {
+        "name": "Essential",
         "fidesDataUseKey": "provide.service",
-        "name": "Provide a service",
-        "url": "https://example.com/privacy#provide-service",
-        "highlight": false,
-        "description": "Manage how we use your data, including Do Not Sell My Personal Information."
+        "default": true,
+        "url": "https://example.com/privacy#essential",
+        "highlight": true,
+        "description": "...",
+        "cookieKeys": ["essential"]
```

Comment:
@mfbrown - This is where I'll need reasonable defaults for the data use keys, their description, and what cookie keys would be generically useful.